scripts/install.sh

```diff
@@ -569,4 +570,7 @@ function main() {
     report_install_success
 }
 
-main "$@"
+# tee logs into /var/log/kurl/install-<date>.log and stdout
+mkdir -p /var/log/kurl
+main "$@" 2>&1 | tee -a /var/log/kurl/install-$(date +"%m-%d-%Y-%H-%M-%S").log
```

Comment:
I think it could be helpful to have the kURL installer ID as part of the filename.

Reply:
Hi @rrpolanco,

> I think it would be helpful to have the kURL installer ID as part of the filename. 

When we will create the log we have not the installedID yet. But we can start to output the spec used for example then we can get it the logs. However, let's leave this one for a follow up? WDYT? Something like "started to output and log the spec". I think is better we try to keep the stories and the PRs shorter as possible. One for each need. 

Hi @banjoh,

> I wonder whether we can echo "$@" as well when the script starts. Logging the entire command would be good.
If we echo it is empty. Note sure what exactly you would like to echo here. 
However, let's it for a follow up as well, if needed. Something like begin to output and log X.

Reply:
> 

I think I wasn't very clear :). I would like to see in the logs any [kurl CLI options](https://kurl.sh/docs/install-with-kurl/advanced-options) a script is invoked with. If there are no options, the list will be empty that's why nothing was printed out when you ran `echo "$@"`. Here is a snippet

```sh
function main() {
    echo "ARGS: $@"
}

main "$@"
```

And after running the script
```sh
$ ./tt.sh container-log-max-files=2 exclude-builtin-host-preflights force-reapply-addons
install.sh container-log-max-files=2 exclude-builtin-host-preflights force-reapply-addons

$ cat tt.sh | bash -s container-log-max-files=2 exclude-builtin-host-preflights force-reapply-addons
install.sh container-log-max-files=2 exclude-builtin-host-preflights force-reapply-addons
```

Reply:
Done 👍